plugins/styleguide/assets/javascripts/discourse/components/styleguide/calendar-date-time-input.js

```diff
@@ -0,0 +1,28 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+
+export default class StyleguideCalendarDateTimeInput extends Component {
+  @service currentUser;
+
+  @tracked dateFormat = "YYYY-MM-DD";
+  @tracked timeFormat = "HH:mm:ss";
+  @tracked date = null;
+  @tracked time = null;
+  @tracked minDate = null;
+
+  constructor() {
+    super(...arguments);
+  }
```

Comment:
I'm curious why this is necessary since we're just calling super without doing anything else.

Reply:
Oh it's probably just leftover cruft, will delete :+1: